broker/cloud_run/lsst/classify_upsilon/main.py

```diff
@@ -0,0 +1,169 @@
+
+    TOPIC.publish(
+        pittgoogle.Alert.from_dict(
+            {**alert_lite.dict, "upsilon": upsilon_dict},
+            attributes={
+                **alert_lite.attributes,
+                "pg_upsilon_u_label": upsilon_dict["u_label"],
+                "pg_upsilon_u_flag": upsilon_dict["u_flag"],
+                "pg_upsilon_g_label": upsilon_dict["g_label"],
+                "pg_upsilon_g_flag": upsilon_dict["g_flag"],
+                "pg_upsilon_r_label": upsilon_dict["r_label"],
+                "pg_upsilon_r_flag": upsilon_dict["r_flag"],
+                "pg_upsilon_i_label": upsilon_dict["i_label"],
+                "pg_upsilon_i_flag": upsilon_dict["i_flag"],
+                "pg_upsilon_z_label": upsilon_dict["z_label"],
+                "pg_upsilon_z_flag": upsilon_dict["z_flag"],
+                "pg_upsilon_y_label": upsilon_dict["y_label"],
+                "pg_upsilon_y_flag": upsilon_dict["y_flag"],
+            },
+            schema_name="default",
+        )
+    )
+
+    return "", HTTP_204
+
+
+def _classify_with_UPSILoN(alert_lite: pittgoogle.Alert) -> Dict:
+    # extract the alert
+    alert_lite_dict = alert_lite.dict["alert_lite"]
+    alert_df = _create_dataframe(alert_lite_dict)
+    bands = ["u", "g", "r", "i", "z", "y"]
+    outgoing_dict = {
+        "diaObjectId": alert_lite_dict["diaObject"]["diaObjectId"],
+        "diaSourceId": alert_lite_dict["diaSource"]["diaSourceId"],
+    }
+
+    # load UPSILoN's classification model
+    rf_model = upsilon.load_rf_model()
+    for band in bands:
+        # define parameters for feature extractions
+        filter_diaSources = alert_df[alert_df["band"] == band]
+        mask = filter_diaSources["psfFlux"].to_numpy() > 0
+        # set output to None if data is absent or there are too few data points for this band; limit set by UPSILoN
+        if filter_diaSources.empty or mask.sum() < 7:
+            outgoing_dict[f"{band}_label"] = None
+            outgoing_dict[f"{band}_probability"] = None
+            outgoing_dict[f"{band}_flag"] = None
+            continue
+
+        flux = filter_diaSources["psfFlux"].to_numpy()[mask]
+        flux_err = filter_diaSources["psfFluxErr"].to_numpy()[mask]
+
+        # UPSILoN requires three features to make a prediction, define them below:
+        date = filter_diaSources["midpointMjdTai"].to_numpy()[mask]
+        mag = _convert_flux_to_mag(flux)
+        mag_err = _calculate_mag_err(flux, flux_err)
+
+        # extract features
+        e_features = upsilon.ExtractFeatures(date, mag, mag_err)
+        e_features.run()
+        features = e_features.get_features()
+
+        # classify
+        label, probability, flag = upsilon.predict(rf_model, features)
+        outgoing_dict[f"{band}_label"] = label
+        outgoing_dict[f"{band}_probability"] = probability
+        outgoing_dict[f"{band}_flag"] = flag
+
+    return outgoing_dict
+
+
+def _create_dataframe(alert_dict: pittgoogle.Alert) -> "pd.DataFrame":
```

Comment:
```suggestion
def _create_dataframe(alert_dict: pittgoogle.Alert) -> pd.DataFrame:
```

No need to quote since you're doing a normal import of pandas.